puppet/zulip/templates/supervisor/zulip.conf.template.erb

```diff
@@ -37,21 +37,19 @@ stdout_logfile_maxbytes=1GB   ; max # logfile bytes b4 rotation (default 50MB)
 stdout_logfile_backups=10     ; # of stdout logfile backups (default 10)
 directory=/home/zulip/deployments/current/
 
-<% @queues.each do |queue| -%>
-[program:zulip_events_<%= queue %>]
-command=/home/zulip/deployments/current/manage.py process_queue --queue_name=<%= queue %>
+[program:zulip_events]
+command=/home/zulip/deployments/current/manage.py process_queue --multi_threaded <%= @queues.join(' ') %>
 priority=300                   ; the relative start priority (default 999)
 autostart=true                 ; start at supervisord start (default: true)
 autorestart=true               ; whether/when to restart (default: unexpected)
 stopsignal=TERM                 ; signal used to kill process (default TERM)
 stopwaitsecs=30                ; max num secs to wait b4 SIGKILL (default 10)
 user=zulip                    ; setuid to this UNIX account to run the program
 redirect_stderr=true           ; redirect proc stderr to stdout (default false)
-stdout_logfile=/var/log/zulip/events_<%= queue %>.log         ; stdout log path, NONE for none; default AUTO
+stdout_logfile=/var/log/zulip/events.log         ; stdout log path, NONE for none; default AUTO
 stdout_logfile_maxbytes=1GB   ; max # logfile bytes b4 rotation (default 50MB)
 stdout_logfile_backups=10     ; # of stdout logfile backups (default 10)
 directory=/home/zulip/deployments/current/
-<% end -%>
```

Comment:
Hmmm...I can't exactly understand the rationale. Multi-process is required only when each processes needs to be sand-boxed (spawned as heavier UNIX subprocesses by supervisord, also note that supervisord already uses multithreading when spawning) at the expense of more RAM. Why should sandboxing be based on whether the system has >4GB of RAM?